content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -160,3 +202,7 @@ OpenTelemetry's [Agent Configuration](https://opentelemetry.io/docs/zero-code/ja
 ## Further reading
 
 {{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://github.com/open-telemetry/opentelemetry-go/tree/main/trace
```

Comment:
FYI these links need to be added to come before Go's `{{% /tab %}}` closing for the links in the Go section to work properly